name.abuchen.portfolio.tests/src/name/abuchen/portfolio/datatransfer/pdf/sbroker/SBrokerPDFExtractorTest.java

```diff
@@ -1767,249 +1767,111 @@ public void testGiroKontoauszug01()
     {
         SBrokerPDFExtractor extractor = new SBrokerPDFExtractor(new Client());
 
-        List<Exception> errors = new ArrayList<Exception>();
+        List<Exception> errors = new ArrayList<>();
 
         List<Item> results = extractor.extract(PDFInputFile.loadTestCase(getClass(), "GiroKontoauszug01.txt"),
                         errors);
 
-        assertThat(errors, empty());
-        assertThat(results.size(), is(20));
-
-        // check transaction
-        // get transactions
-        Iterator<Extractor.Item> iter = results.stream().filter(TransactionItem.class::isInstance).iterator();
-        assertThat(results.stream().filter(TransactionItem.class::isInstance).count(), is(20L));
-
-        Item item = iter.next();
+//        assertThat(errors, empty());
+//        assertThat(countSecurities(results), is(0L));
+//        assertThat(countBuySell(results), is(0L));
+//        assertThat(countAccountTransactions(results), is(20L));
+//        assertThat(results.size(), is(20));
+//        new AssertImportActions().check(results, CurrencyUnit.EUR);
```

Comment:
Commented out code.

Reply:
I had tested something and forgot to activate it again... thanks!
Done.